.env.example

```diff
@@ -12,3 +12,7 @@ ADYEN_ORIGIN_KEY=pub.v2.something-kinda-secret
 ADYEN_ENVIRONMENT=<test|live>
 
 APP_ENVIRONMENT=<development|staging|production>
+
+# Optional - if you want to test uploading source maps to Sentry
+HEROKU_SLUG_COMMIT=<commit hash>
+SENTRY_AUTH_TOKEN=<sentry auth token>
```

Comment:
Maybe the `SENTRY_AUTH_TOKEN` should also be added to this file?

Reply:
What do you mean? It's in the file, no?

Reply:
Hahaha what?? I'm blind apparently? That's kind of scary actually